ckanext/s3filestore/uploader.py

```diff
@@ -148,10 +140,13 @@ def get_signed_url_to_key(self, key_path, expiredin=60):
         # check whether the object exists in S3
         client.head_object(Bucket=self.bucket_name, Key=key_path)
 
-        return client.generate_presigned_url(ClientMethod='get_object',
+        url = client.generate_presigned_url(ClientMethod='get_object',
                                             Params={'Bucket': self.bucket_name,
                                                     'Key': key_path},
                                             ExpiresIn=expiredin)
```

Comment:
will time locking be happening in a later chg?

Reply:
That's the plan.